deployments/helm/nvidia-device-plugin/values.yaml

```diff
@@ -145,6 +145,7 @@ nfd:
           - vendor
 
 mps:
+  enabled: true
```

Comment:
As a user, I think at a minimum there needs to be comments placed above this line to explain its use. If I see this in a values file, I may interpret `mps.enabled: true` to mean I, as a user, want MPS enabled right out of the gate. But that's not what this does. When set to `false` MPS will never work even if I later activate an MPS configuration by labeling a node.

Reply:
I added a comment clarifying this